content/robertstettner/drone-cloudformation/index.md

```diff
@@ -0,0 +1,111 @@
+---
+date: 2017-07-18T00:00:00+00:00
+title: Amazon CloudFormation
+author: robertstettner
+tags: [ cloudformation, aws, amazon ]
+repo: robertstettner/drone-cloudformation
+logo: amazon_cloudformation.svg
+image: robertstettner/drone-cloudformation
+---
+The CloudFormation plugin can be used to create/update or delete stacks, and to validate templates. The below pipeline configuration demonstrates simple deployment usage:
+
+```yaml
+pipeline:
+  deploy:
+    image: robertstettner/drone-cloudformation
+    stackname: my-awesome-stack-${DRONE_BRANCH}
+    template: templates/stack.yml
+    params:
+      Version: "${DRONE_BUILD_NUMBER}"
+      Environment: ${DRONE_DEPLOY_TO}
```

Comment:
can you remove the `${VARIABLE}` notation from the docs and use dummy values? In the event we ever change variable names, this prevents your documentation from becoming outdated.